webapp/components/admin_console/email_settings.jsx

```diff
@@ -11,11 +11,15 @@ import * as Utils from 'utils/utils.jsx';
 import AdminSettings from './admin_settings.jsx';
 import BooleanSetting from './boolean_setting.jsx';
 import {ConnectionSecurityDropdownSettingEmail} from './connection_security_dropdown_setting.jsx';
+import DropdownSetting from './dropdown_setting.jsx';
 import EmailConnectionTest from './email_connection_test.jsx';
 import {FormattedHTMLMessage, FormattedMessage} from 'react-intl';
 import SettingsGroup from './settings_group.jsx';
 import TextSetting from './text_setting.jsx';
 
+const EMAIL_NOTIFICATION_CONTENTS_FULL = 'full';
+const EMAIL_NOTIFICATION_CONTENTS_GENERIC = 'generic';
```

Comment:
These would usually go in `utils/constants.jsx` although that file is incredibly bloated as is. I'd be fine leaving them here since nowhere else on the client care about these settings.